packages/lab/stories/layout/flow-layout.stories.tsx

```diff
@@ -52,27 +62,203 @@ const Responsive: ComponentStory<typeof FlowLayout> = (args) => {
 export const ToolkitFlowLayoutResponsive = Responsive.bind({});
 ToolkitFlowLayoutResponsive.args = {};
 
+const FormStep = () => (
+  <StackLayout>
+    <p className="form-step-label">Label</p>
+    <h1>Value detail</h1>
+  </StackLayout>
+);
+
 const FlowLayoutStorySimpleUsage: ComponentStory<typeof FlowLayout> = (
   args
 ) => {
   return (
-    <div style={{ background: "#EAEDEF", padding: 24 }}>
-      <FlowLayout {...args}>
-        {Array.from({ length: 5 }, (_, index) => (
-          <Metric key={index}>
-            <MetricHeader title={`Form Stage ${index + 1}`} />
-            <MetricContent value="Complete" />
-          </Metric>
+    <div className="uitkEmphasisHigh flow-layout-container">
+      <FlowLayout gap={4} {...args}>
+        {Array.from({ length: 6 }, (_, index) => (
+          <FormStep key={index} />
         ))}
-        <Metric>
-          <MetricHeader title="Form Stage 6" />
-          <MetricContent value="Pending" />
-        </Metric>
       </FlowLayout>
     </div>
   );
 };
+
 export const FlowLayoutSimpleUsage = FlowLayoutStorySimpleUsage.bind({});
-FlowLayoutSimpleUsage.args = {
+
+const dropdownExampleData = ["No", "Yes"];
+
+const RightForm = () => (
+  <Accordion>
+    {Array.from({ length: 6 }, (_, index) => (
+      <AccordionSection key={index} defaultExpanded={index === 0}>
+        <AccordionSummary>Expandable and collapsible section</AccordionSummary>
+        <AccordionDetails>
+          <StackLayout>
```

Comment:
I would move this to stackLayout simple usage, and then import it here for the composite, that ways we have examples there too